src/application.ts

```diff
@@ -32,11 +32,18 @@ export class HelloWorldApp extends RepositoryMixin(Application) {
     server
       .bind(AuthenticationBindings.STRATEGY)
       .toProvider(MyAuthStrategyProvider);
-    await super.start();
-    let port = await server.get(RestBindings.PORT);
-    console.log(`HTTP server listening on port ${port}`);
-    console.log(
-      `Run \'curl localhost:${port}/helloworld?name=YOUR_NAME\' to try it out`,
+    await super.start().then(
+      async () => {
+        let port = await server.get(RestBindings.PORT);
+        console.log(`HTTP server listening on port ${port}`);
+        console.log(
+          `Run \'curl localhost:${port}/helloworld?name=YOUR_NAME\' to try it out`,
+        );
+      },
+      err => {
```

Comment:
A convention question: do we have any code convention about using 2nd param of `then()` or use `catch()` for error?

Reply:
Good question. I should look through our code base to find a pattern!